benches/benches/block_target_gas_set/crypto.rs

```diff
@@ -0,0 +1,147 @@
+use crate::{
+    utils::generate_linear_costs,
+    *,
+};
+use rand::{
+    rngs::StdRng,
+    SeedableRng,
+};
+
+// ECK1: Secp251k1 signature recovery
+// ECR1: Secp256r1 signature recovery
+// ED19: edDSA curve25519 verification
+// K256: keccak-256
+// S256: SHA-2-256
+pub fn run_crypto(group: &mut BenchmarkGroup<WallTime>) {
+    let rng = &mut StdRng::seed_from_u64(2322u64);
+
+    let message = Message::new(b"foo");
```

Comment:
The `op::eck1` expects 32 bytes as a message, so it is better to use `Bytes32` to make it more clear.